cli/azd/pkg/project/project_config.go

```diff
@@ -28,10 +28,16 @@ type ProjectConfig struct {
 	Metadata          *ProjectMetadata          `yaml:"metadata,omitempty"`
 	Services          map[string]*ServiceConfig `yaml:",omitempty"`
 	Infra             provisioning.Options      `yaml:"infra"`
+	Pipeline          PipelineOptions           `yaml:"pipeline"`
 
 	handlers map[Event][]ProjectLifecycleEventHandlerFn
 }
 
+// options supported in azure.yaml
+type PipelineOptions struct {
+	Provider string `yaml:"provider"`
+}
+
```

Comment:
Don't forget to update the JSON schema for any azure.yaml schema updates.

Reply:
I think I missed one more commit with the change to azure.yaml.  Will push it now. Thanks

Reply:
done